web/vital_records/mixins.py

```diff
@@ -6,6 +6,14 @@
 from web.vital_records.session import Session
 
 
+class DisableFieldsMixin:
+    def __init__(self, *args, **kwargs):
+        super(DisableFieldsMixin, self).__init__(*args, **kwargs)
+        if hasattr(self, "instance") and self.instance is not None and self.instance.already_submitted:
+            for field_name, field in self.fields.items():
```

Comment:
Nice, good change. 

Since `self.fields` is (appears to be) a `dict`, and we aren't using `field_name`, maybe just loop over `.values()`? Not a big deal either way.